jobs/inprogress_update/inprogress_update.py

```diff
@@ -93,9 +93,30 @@ def inprogress_update(user: User, max_rows: int, client_delay: int, examine_dela
                 event = Event.MARKED_ON_HOLD
 
             db.session.add(r)
+            db.session.commit()
             EventRecorder.record(user, event, r, r.json(), save_to_session=True)
 
-        db.session.commit()
+        # for NRs showing in NRO_UPDATING status need to be set to DRAFT
+        nro_updating_reqs = db.session.query(Request). \
+            filter(Request.stateCd == State.NRO_UPDATING). \
+            order_by(Request.lastUpdate.asc()). \
+            limit(max_rows). \
+            with_for_update().all()
+
+        for r in nro_updating_reqs:
+            row_count += 1
+            
+            current_app.logger.debug(f'processing nr: {r.nrNum}, state: {r.stateCd}, previous state: {r.previousStateCd}, last_update: {r.lastUpdate}')    
+            
+            if r.previousStateCd == None:
+                r.stateCd = State.DRAFT
```

Comment:
you are updating the state here and I know you were running into some issues with the queue related issues.  I imagine that was happening because the `Request` model has a listener that results in a cloud event being published.  Were you able confirm that works locally?  None of the queue configuration or queue initialization code seems to be in place for this job.  I imagine you will run into issues if you deploy this to DEV.

Reply:
@argush3  I have Rocket.Chat issue today. I will talk to you detail after I get rc running.